feat(bullmq): allow control over log level

LonguCodes opened the pull request:
## Information

| Type                  | Breaking change |
| --------------------- | --------------- |
| Feature | No          |

---


## Usage example

Example to use your feature and to improve the documentation after merging your PR:
```typescript
@Configuration({
  bullmq: {
    // ...
    logLevel: "off"
    // ...
  }
})
export class Server {}


```

## Todos

- [ ] Tests - N/A
- [ ] Coverage - N/A
- [x] Example
- [x] Documentation


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **New Features**
  * Configurable logging for the BullMQ integration via a new logLevel option (debug, info, warn, error, off, all); job processing uses this setting and defaults to info.

* **Documentation**
  * README updated with a Logs section describing default behavior, available log levels, examples (including disabling logs), and a note that missing job controllers emit warnings.
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

LonguCodes committed:
feat(bullmq): allow control over log level

LonguCodes commented:
@Romakita I did not need help before, but now I do :/  Can you check what's up with the instalation error? I did not change anything

Romakita commented:
@LonguCodes fixed... I don't know why but dependencies haven't been correctly fixed in the yarn.lock on the production branch. I fixed the production branch and rebased your branch to my origin/production.

Romakita committed:
docs: update packages/third-parties/bullmq/README.md

Co-authored-by: coderabbitai[bot] <136622811+coderabbitai[bot]@users.noreply.github.com>

Romakita commented:
Let's go for merge ;) @LonguCodes thanks!

Romakita commented:
:tada: This PR is included in version 8.17.0 :tada:

The release is available on:
- `v8.17.0`
- [GitHub release](https://github.com/tsedio/tsed/releases/tag/v8.17.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:

LonguCodes commented:
> @LonguCodes fixed... I don't know why but dependencies haven't been correctly fixed in the yarn.lock on the production branch. I fixed the production branch and rebased your branch to my origin/production.
> 
> Let me know when your PR is ready to review

Sorry for not responding, real busy. Thanks for merging!
